pkg/querier/blocks_store_queryable_test.go

```diff
@@ -1928,6 +1929,14 @@ func mockSeriesResponseWithChunks(lbls labels.Labels, chunks ...storepb.AggrChun
 	}
 }
 
+func mockStatsResponse(stat int) *storepb.SeriesResponse {
```

Comment:
[nit] I would clarify it.

```suggestion
func mockStatsResponse(fetchedIndexBytes int) *storepb.SeriesResponse {
```